vmhost/contexts/storage.go

```diff
@@ -524,8 +524,17 @@ func (context *storageContext) GetStorageLoadCost(trieDepth uint32, staticGasCos
 }
 
 func computeGasForStorageLoadBasedOnTrieDepth(trieDepth int64, coefficients config.DynamicStorageLoadCostCoefficients, staticGasCost uint64) (uint64, error) {
-	// TODO protect computations against overflow
-	fx := coefficients.Quadratic*trieDepth*trieDepth + coefficients.Linear*trieDepth + coefficients.Constant
+	quadraticTerm := math.MulInt64(
```

Comment:
Missing `TestStorageContext_GetStorageLoadCost` tests from the PR targeted to `rc/v1.6`

Reply:
Added; please confirm.